### ISSUE TITLE
Accessibility: Landmarks should have (hidden) headings

### ISSUE THREAD

**Issue by pjeweb** [alveusgg/alveusgg#154]
- The page header (`<header>`) should have a hidden heading
- Each `<section>` should have a meaningful heading or it can just be a `<div>`. Especially the section around the whole page does not make much sense and makes it so the page header and footer are scoped to the section and not the whole page.
- The page footer (`<footer>`) should have a hidden heading

Hide heading, but keep for screen readers with `sr-only` (tailwind).
